knr1050Sup/knr1050.db

```diff
@@ -46,48 +47,26 @@ record(calcout, "$(P)DISABLE:CHECK") {
 # /// can be remotely executed accept for switching to remote
 # /// and checking the current remote/local state. If in remote
 # /// mode then remotely executed commands are accepted but the
-# /// front panel of the device is unavilable.
+# /// front panel of the device is unavailable.
 # ///
-record(bi, "$(P)TOGGLE:REMOTE:SP") {
-	field(DESC, "Toggle remote/local mode")
-	
-	field(ZNAM, "Local")
-	field(ONAM, "Remote")
-	info(INTEREST, "HIGH")
-}
-
-# ///
-# /// For the given toggle pump set point process the desired record.
-# ///
-record(calcout, "$(P)_TOG:CALC:REMOTE") {
-	field(INPA, "$(P)TOGGLE:REMOTE:SP CP")
-	
-	field(CALC, "A+1")
-	
-	field(OUT, "$(P)_TOG:REMOTE.SELN PP")
-}
-
-record(fanout, "$(P)_TOG:REMOTE") {
-	field(SELM, "Specified")
-	
-	field(LNK1, "$(P)_LOCAL_MODE:SP PP")
-	field(LNK2, "$(P)_REMOTE_MODE:SP PP")
-}
-
-record(bo, "$(P)_REMOTE_MODE:SP") {
-	field(DESC, "Set the device into remote/local mode")
+record(bo, "$(P)REMOTE_MODE:SP") {
+	field(DESC, "Set the device into remote mode")
 	field(DTYP, "stream")
 	field(SCAN, "Passive")
 	
 	field(OUT, "@knr1050.proto set_remote_mode() $(PORT)")
+	field(ZNAM, "")
+	field(ONAM, "Remote")
 }
 
-record(bo, "$(P)_LOCAL_MODE:SP") {
+record(bo, "$(P)LOCAL_MODE:SP") {
 	field(DESC, "Sets the device into local mode")
 	field(DTYP, "stream")
 	field(SCAN, "Passive")
 	
 	field(OUT, "@knr1050.proto set_local_mode() $(PORT)")
+	field(ZNAM, "")
+	field(ONAM, "Local")
 }
 
 record(bi, "$(P)GET_REMOTE_MODE") {
```

Comment:
Our standards for PV naming (https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming) would probably just call this `STATE` or `MODE`. Generally, PVs are named for what data they represent, no the action they perform while functions in the protocol file are named by the action they perform.